bitshuffle.py

```diff
@@ -27,6 +28,10 @@
 stdin = sys.stdin
 
 
+# Change this version variable to change the --version output
+program_version = 1.0
```

Comment:
Already addressed in https://github.com/charlesdaniels/bitshuffle/commit/d539f92f80ed6986adb90bb289661878d90f4370#diff-8285d39e0f32f542d185e03a08248293